client/src/app/css/style.scss

```diff
@@ -156,8 +159,17 @@ main {
   border-radius: 4px;
   background-color: rgba(0, 0, 0, 0.80);
   padding: 40px;
+  font-size: 2rem !important;
```

Comment:
try to avoid using `!important`

Reply:
materialize definitions make it difficult to override so on this occasion this is necessary. In planning for the next project i'll be moving away from materialize.